zhaquirks/philips/hue_light.py

```diff
@@ -4,6 +4,19 @@
 
 from zhaquirks.philips import PHILIPS, SIGNIFY, PhilipsHueLightCluster
 
+(
+    QuirkBuilder()
+    .applies_to(SIGNIFY, "929004608001")
+    .applies_to(SIGNIFY, "929004608101")
+    .applies_to(SIGNIFY, "929004608201")
+    .friendly_name(
+        model="Hue OmniGlow Lightstrip",
```

Comment:
To follow existing conventions, let's lower-case "lightstrip".
```suggestion
        model="Hue OmniGlow lightstrip",
```